pom.xml

```diff
@@ -2,8 +2,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>no.template</groupId>
-    <artifactId>template-api</artifactId>
+    <groupId>no.brreg.abackendservice</groupId>
```

Comment:
Burde abackendservice være en del av groupid?

Reply:
Jeg tenker at det hadde vært greit å alltid bare bruke "no.brreg" som groupid, men det er vel strengt tatt opp til @stigbd

Reply:
Hmm... Eg er usikker på kva konsekvensen av det eine eller andre er. Kva er eigentlig semantikken til groupid?

Reply:
https://maven.apache.org/guides/mini/guide-naming-conventions.html

Reply:
Vi burde kanskje ha en samlende groupid som ikke er direkte no.brreg?

no.brreg.fdk? no.brreg.api? no.brreg.fdk.api?

Reply:
Det viktigaste er i alle fall å fjerne "template". 
A-backend-service er ikkje knytta til noko spesifikt domene i no.brreg, heller ikkje fdk. Kva vil a-front-end-service få?
Samtitig er no.brreg "for stort".
Slik sett er følgande beste kandidat:
no.brreg.informasjonsforvaltning
(trur eg)

Reply:
Nå er groupid "no.brreg.informasjonsforvaltning" og pakken "no.brreg.informasjonsforvaltning.abackendservice"